src/main/java/xyz/duncanruns/julti/util/LauncherUtil.java

```diff
@@ -0,0 +1,33 @@
+package xyz.duncanruns.julti.util;
+
+import static xyz.duncanruns.julti.Julti.log;
+
+import java.awt.Desktop;
+import java.io.IOException;
+
+import org.apache.logging.log4j.Level;
+
+import xyz.duncanruns.julti.JultiOptions;
+
+public class LauncherUtil {
+    private LauncherUtil() {
+    }
+
+    public static void launchPrograms() {
+        JultiOptions options = JultiOptions.getJultiOptions();
+
+        if (!Desktop.isDesktopSupported()) {
+            log(Level.ERROR, "Could not launch programs: Java Desktop not supported.");
+            return;
+        }
+
+        options.launchingProgramPaths.forEach(path -> {
+            try {
+                // FIXME: This fails with programs like OBS because it opens them from directory julti is being ran, not their parent directory
```

Comment:
Generally PRs shouldn't include //FIXMEs or //TODOs

Anyway I am a fan of this `getDesktop().open()` method so we can try to see if this can be modified to work.
The method's body is:
```java
        file = new File(file.getPath());
        checkAWTPermission();
        checkExec();
        checkActionSupport(Action.OPEN);
        checkFileValidation(file);

        peer.open(file);
```
`peer.open(file);` (on windows) will run the `private static native String ShellExecute(String, String)` method

It looks like with the existing Java abstractions we wouldn't be able to adjust this to run in the directory we want. However the native code it will be running must be some sort of accessible function.

And it didn't take me long at all to find <https://learn.microsoft.com/en-us/windows/win32/api/shellapi/nf-shellapi-shellexecutea>
It also takes more parameters than Java will use for `getDesktop().open()`, notably `lpDirectory`:
> A pointer to a null-terminated string that specifies the default (working) directory for the action.

We would probably end up with a line of code that looks like this:
```java
Shell32.INSTANCE.ShellExecute(null,"open",path.toString(),null,path.getParent().toString(),1);
```
Although we might have to redefine this ourselves because we might need it to be `ShellExecuteA` instead of just `ShellExecute` for janky Java Windows API reasons.

- The 1st parameter is a window, but only for window related actions so null.
- The 2nd parameter is the action, and we want to open, so "open".
- The 3rd parameter is the file, so we pass the path as a string.
- The 4th parameter is arguments for execution, this is mainly for executables, but we don't want to do anything fancy and just plain opening with no args is fine, so null.
- The 5th parameter is `lpDirectory` and this is the bit we actually want, so we go ahead and put the parent directory.
- The 6th parameter is for how the window will show up, I just put 1 for "normal".